llvm/cmake/modules/AddLLVM.cmake

```diff
@@ -314,6 +308,11 @@ function(add_link_opts target_name)
     set_property(TARGET ${target_name} APPEND_STRING PROPERTY
                  LINK_FLAGS " -Wl,-brtl")
   endif()
+
+  if (NOT MINGW)
+    llvm_check_linker_flag(CXX "-Wl,-Bsymbolic-functions"
+                           LINKER_SUPPORTS_B_SYMBOLIC_FUNCTIONS)
+  endif()
```

Comment:
This `if` is unnecessary thanks to this PR.

Reply:
That didn't make any sense to me. Which ```if``` and which PR?

Reply:
I wasn't sure about that and left MinGW as is but it looks like it's just using either LLD or the binutils BFD linker depending on which combo compiler / toolchain you go with.

Reply:
BFD accepts `-Bsymbolic-functions` but it's a no-op for anything other than ELF.  LLD rejects the option as unknown in the MinGW driver.  So it would be fine to remove the `if (NOT MINGW)` wrapping that, and let it use the flag or not depending on whether the linker accepts it.